spec/models/saved_claim/caregivers_assistance_claim_spec.rb

```diff
@@ -33,56 +31,67 @@
     end
 
     it 'calls PdfFill::Filler#fill_form' do
-      if RUBY_VERSION =~ /2.7/
-        expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid, {}).once.and_return(:expected_file_paths)
-      else
-        expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid).once.and_return(:expected_file_paths)
-      end
+      expect(PdfFill::Filler).to receive(:fill_form).with(claim, claim.guid).once.and_return(:expected_file_paths)
       expect(claim.to_pdf).to eq(:expected_file_paths)
     end
 
-    it 'passes arguments to PdfFill::Filler#fill_form' do
-      if RUBY_VERSION =~ /2.7/
```

Comment:
Removed this condition in the spec that was added [here](https://github.com/department-of-veterans-affairs/vets-api/pull/11929) since we are no longer on ruby 2.7.